src/coprocessor/util.rs

```diff
@@ -134,6 +134,60 @@ pub fn get_pk(col: &ColumnInfo, h: i64) -> Datum {
     }
 }
 
+use std::time::Duration;
+
+// RuntimeStats collects one executor's execution info.
+#[derive(Debug, Default)]
+pub struct RuntimeStats {
+    // executor consume time.
+    pub consume: Duration,
+    // executor's Next() called times.
+    pub count: i64,
+    // executor return row count.
+    pub rows: i64,
+}
+
+#[allow(dead_code)]
```

Comment:
Please remove "allow(dead_code)" and remove unnecessary functions in Rust. You can implement it in a more Rustic way and not following TiDB's impl.